tutorecommerce/templates/ecommerce/apps/ecommerce/settings/partials/common.py

```diff
@@ -91,6 +91,8 @@
 PAYMENT_PROCESSOR_CONFIG = {
     "openedx": common_payment_processor_config,
     "dev": common_payment_processor_config,
+    "edx" : common_payment_processor_config,
+    "other" : common_payment_processor_config,
```

Comment:
Why did you add these new payment processors?

Reply:
<img width="1356" alt="Screenshot 2023-06-01 at 3 19 12 PM" src="https://github.com/overhangio/tutor-ecommerce/assets/122095701/92c5a3a9-431d-4867-9034-17cb6f13c34c">

I was getting an error due to no key "edx".